Projects/Claims/Sources/Views/SubmitClaim/SingleItem/SubmitClaimSingleItem.swift

```diff
@@ -7,147 +7,124 @@ import hGraphQL
 public struct SubmitClaimSingleItem: View {
     @PresentableStore var store: SubmitClaimStore
     @State var purchasePrice: String = ""
+    @State var validPriceInput = false
+    @State var type: ClaimsFlowSingleItemFieldType?
 
     public init() {}
-
+    
     public var body: some View {
         LoadingViewWithContent(.postSingleItem) {
             hForm {
-                PresentableStoreLens(
-                    SubmitClaimStore.self,
-                    getter: { state in
-                        state.singleItemStep
-                    }
-                ) { singleItemStep in
-                    displayBrandAndModelField(singleItemStep: singleItemStep)
-                    displayDateField(claim: singleItemStep)
-                    displayPurchasePriceField(claim: singleItemStep)
-                    displayDamageField(claim: singleItemStep)
-                }
             }
+            .hUseNewStyle
+            .hFormTitle(.small, L10n.claimsSingleItemDetails)
             .hFormAttachToBottom {
-                hButton.LargeButtonFilled {
-                    store.send(.singleItemRequest(purchasePrice: Double(purchasePrice)))
-                    UIApplication.dismissKeyboard()
-                } content: {
-                    hText(L10n.generalContinueButton)
+                
+                VStack(spacing: 8) {
+                    PresentableStoreLens(
+                        SubmitClaimStore.self,
+                        getter: { state in
+                            state.singleItemStep
+                        }
+                    ) { singleItemStep in
+                        displayBrandAndModelField(singleItemStep: singleItemStep)
+                        displayDateField(claim: singleItemStep)
+                        displayPurchasePriceField(claim: singleItemStep)
+                        displayDamageField(claim: singleItemStep)
+                        NoticeComponent(text: L10n.claimsSingleItemNoticeLabel)
+                    }
+                    
+                    hButton.LargeButtonFilled {
+                        store.send(.singleItemRequest(purchasePrice: Double(purchasePrice)))
+                        UIApplication.dismissKeyboard()
+                    } content: {
+                        hText(L10n.generalContinueButton)
+                    }
+                    .padding([.leading, .trailing], 16)
+                    .padding(.bottom, 8)
                 }
-                .padding([.leading, .trailing], 16)
-                .padding(.bottom, 6)
             }
         }
```

Comment:
...... to here  .hUseNewStyle

On this way any elements will have hUseNewStyle environment property set properly if used